homeassistant/components/home_connect/coordinator.py

```diff
@@ -0,0 +1,226 @@
+                                    event.key not in StatusKey
+                                    or event.key is EventKey.UNKNOWN
+                                ):
+                                    continue
+                                status_key = StatusKey(event.key)
+                                if status_key in statuses:
+                                    statuses[status_key].value = event.value
+                                else:
+                                    statuses[status_key] = Status(
+                                        status_key, event.value
+                                    )
+                            await self._call_event_listener(event_message)
+
+                        case EventType.NOTIFY:
+                            settings = self.data[event_message.ha_id].settings
+                            events = self.data[event_message.ha_id].events
+                            for event in event_message.data.items:
+                                if event.key is EventKey.UNKNOWN:
+                                    continue
+                                if event.key in SettingKey:
+                                    setting_key = SettingKey(event.key)
+                                    if setting_key in settings:
+                                        settings[setting_key].value = event.value
+                                    else:
+                                        settings[setting_key] = GetSetting(
+                                            setting_key, event.value
+                                        )
+                                else:
+                                    events[event.key] = event
+                            await self._call_event_listener(event_message)
+
+                        case EventType.EVENT:
+                            events = self.data[event_message.ha_id].events
+                            for event in event_message.data.items:
+                                events[event.key] = event
+                            await self._call_event_listener(event_message)
+
+            except (EventStreamInterruptedError, HomeConnectRequestError) as error:
+                _LOGGER.debug(
+                    "Non-breaking error (%s) while listening for events,"
+                    " continuing in 30 seconds",
+                    type(error).__name__,
+                )
+                await asyncio.sleep(EVENT_STREAM_RECONNECT_DELAY)
+            except HomeConnectApiError as error:
+                _LOGGER.error("Error while listening for events: %s", error)
+                self.hass.config_entries.async_schedule_reload(
+                    self.config_entry.entry_id
+                )
+                break
+
+    async def _call_event_listener(self, event_message: EventMessage):
+        """Call listener for event."""
+        for event in event_message.data.items:
+            if event.key is EventKey.UNKNOWN:
```

Comment:
Do we really need this case? There would need to be a listener registered for this key before it would matter. I don't think we will register a listener for this key, and if we would register a listener for this key, we probably want to see those events.